src/arguments/default/cls-expr-meth.template

```diff
@@ -15,7 +15,7 @@ info: |
 
 var callCount = 0;
 var C = class {
-  method(/*{ params }*/) {
```

Comment:
on the arguments folder, there's no case providing `/*{ params }*/` that's what I removed in there.